api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetUpStabilityAssessmentReport.java

```diff
@@ -158,7 +161,7 @@ public ReportDefinition constructReportDefinition() {
 
     @Override
     public String getVersion() {
-        return "3.1.0";
+        return "6.0.3";
```

Comment:
Why is there a big Jump on versions from 3.1.0 to 6.0.3

Reply:
Thats because of the number of times i have been testing, remember every testing has a different version

Reply:
@solemabrothers Remember the version is targeted towards health centres changes that are out there not your development testings.

Reply:
Report Version has been changed to 3.2.0